daktari/checks/intellij_idea.py

```diff
@@ -84,6 +84,29 @@ def get_intellij_idea_version_tarball() -> Optional[VersionInfo]:
         return None
 
     product_info_path = os.path.join(os.path.dirname(idea_bin_path), "..", "product-info.json")
+    return get_intellij_version_from_product_info(product_info_path)
+
+
+def get_intellij_idea_toolbox_version() -> Optional[VersionInfo]:
+    try:
+        idea_bin_path = run_command(["sh", "-c", "which idea"]).stdout.rstrip("\n")
+    except CommandErrorException:
+        logging.debug("Could not locate idea", exc_info=True)
+        return None
+
+    apps_dir = os.path.join(os.path.dirname(idea_bin_path), "..", "apps")
+    toolbox_apps = os.listdir(apps_dir)
+    logging.debug(f"Toolbox apps found: {toolbox_apps}", exc_info=True)
+    intellij_installs = [app for app in toolbox_apps if "intellij-idea" in app]
+    if len(intellij_installs) == 0:
```

Comment:
Mine's called `intellij-idea-ultimate` FWIW, but went for a weaker check in the hopes this would also catch the community edition.